packages/eas-cli/src/build/local.ts

```diff
@@ -60,6 +61,8 @@ export async function runLocalBuildAsync(
       ...env,
       ...process.env,
       EAS_LOCAL_BUILD_WORKINGDIR: options.workingdir ?? process.env.EAS_LOCAL_BUILD_WORKINGDIR,
+      __API_SERVER_URL: getExpoApiGraphqlUrl(),
```

Comment:
```suggestion
      __API_SERVER_URL: getExpoApiBaseUrl(),
```

https://github.com/expo/universe/blob/823cfc36d623cc5f8d62e222576c12c154e27f92/server/www/k8s/production/www.env#L37 -> https://github.com/expo/universe/blob/5aff3d6e18cc68a0c253f945823ab8d9b7449b31/server/www/src/Env.ts#L62 -> https://github.com/expo/universe/blob/a9d0e51eb7512befc6bc9fc8d78326d40f8ac467/server/www/src/data/entities/turtle-job-run/TurtleJobRunUtils.ts#L160